coral-hive/build.gradle

```diff
@@ -35,6 +37,28 @@ dependencies {
   testCompile deps.'kryo'
 }
 
+sourceSets.main.antlr.srcDirs = ['src/main/antlr/roots']
+
+generateGrammarSource {
+  maxHeapSize = "4g"
+  arguments += ['-lib', 'src/main/antlr/imports']
+}
+
+task copyAntlrGeneratedSourceFiles(type: Copy) {
```

Comment:
The `generateGrammarSource` task automatically adds the generated files to the compile sourceSets, so the `copyAntlrGeneratedSourceFiles` and `deleteAntlrGeneratedSourceFiles` tasks are unnecessary (unless I'm missing something).

Reply:
Removed those additional tasks. Original intent was to make it easy to discover the generated code, and not have to refresh Gradle in the IDE to pick up the new files and avoid unresolved classes, but I think it is fair to do that when there is generated code.